bindings/tls.c

```diff
@@ -92,7 +92,7 @@ solo5_result_t solo5_tls_init(uintptr_t tls)
 
     /* set tp at its proper place in the TLS block */
     uintptr_t *tmp = (uintptr_t*)solo5_tls_tp_offset(tls);
-    *tmp = *tmp;
```

Comment:
Oops, no dereferencing here, it should have been `*tmp = (uintptr_t)tmp;` or something like that if we want to avoid a second call for `solo5_tls_tp_offset(.)`.

Reply:
CI seems to be fine with that update.